azext_iot/_help.py

```diff
@@ -576,6 +576,49 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state"
+] = """
+    type: group
+    short-summary: Download, upload or migrate the state of an IoT Hub.
```

Comment:
Manage the state of an IoT Hub

we want to make sure the wording would allow extra features if we decide to add them (ex: reset an iot hub)